staging/src/k8s.io/kube-controller-manager/features/kube_controller_manager_features.go

```diff
@@ -0,0 +1,38 @@
+package features
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// Every feature gate should add method here following this template:
+	//
+	// // owner: @username
+	// // alpha: v1.4
+	// MyFeature() bool
+
+	// owner: TBD
+	// alpha: TBD
+	// beta: TBD
+	// deprecated: TBD
+	//
+	// CronjobController2 controls whether the controller manager starts old cronjob
+	// controller or new one which is implemented with informers and delaying queue
+	//
+	// This feature is deprecated, and will be removed in v1.22.
+	CronjobController2 featuregate.Feature = "CronjobController2"
```

Comment:
I would make the feature gate name contain the resource name (CronJob).
eg: `UseCronJobControllerVersion2`, `AlternativeCronJobController`

Reply:
Yeah, that makes sense I'd suggest `CronJobControllerV2`

Reply:
+1, changed to `CronJobControllerV2`